server/docker.go

```diff
@@ -3,34 +3,22 @@ package server
 import (
 	"context"
 	"fmt"
-	"net"
 	"strconv"
 	"strings"
 	"sync"
 	"time"
 
 	dockertypes "github.com/docker/docker/api/types"
-	"github.com/docker/docker/api/types/filters"
-	"github.com/docker/docker/api/types/swarm"
-	swarmtypes "github.com/docker/docker/api/types/swarm"
-	"github.com/docker/docker/api/types/versions"
+	"github.com/docker/docker/api/types/container"
 	"github.com/docker/docker/client"
 	"github.com/sirupsen/logrus"
 )
 
 type IDockerWatcher interface {
-	StartInSwarm(timeoutSeconds int, refreshIntervalSeconds int) error
+	Start(timeoutSeconds int, refreshIntervalSeconds int) error
 	Stop()
 }
 
-var DockerWatcher IDockerWatcher = &dockerWatcherImpl{}
-
-type dockerWatcherImpl struct {
-	sync.RWMutex
-	client        *client.Client
-	contextCancel context.CancelFunc
-}
-
 const (
 	DockerConfigHost         = "unix:///var/run/docker.sock"
```

Comment:
It would be nice for this to be configurable, such as for local testing on Windows...but I don't feel strongly about.

Reply:
Done!